scripts/colour-studio/src/index.js

```diff
@@ -417,28 +440,44 @@ function renderLeonardoHtml(leonardoConfig) {
   return html;
 }
 
-const keyColorsHtml = renderKeyColorsHtml(leonardoConfig);
-const themesJson = generateThemesJson(leonardoConfig);
-const themesHtml = renderThemesHtml(themesJson);
-const themesHtmlAlpha = renderThemesHtml(themesJson, true);
-const tokensStudioHtml = renderTokensStudioHtml(themesJson);
-const cssVars = createCssVars(themesJson);
-const leonardoHtml = renderLeonardoHtml(leonardoConfig);
+let themesJson = {};
+let themesHtml = {};
 
-document.querySelector(".key-colors").innerHTML = keyColorsHtml;
+function renderPage() {
+  themesJson = generateThemesJson(leonardoConfig);
+  themesHtml = renderThemesHtml(themesJson);
 
-document.querySelector(".themes").innerHTML =
-  Object.values(themesHtml).join("");
+  const keyColorsHtml = renderKeyColorsHtml(leonardoConfig);
+  const themesHtmlAlpha = renderThemesHtml(themesJson, true);
+  const tokensStudioHtml = renderTokensStudioHtml(themesJson);
+  const cssVars = createCssVars(themesJson);
+  const leonardoHtml = renderLeonardoHtml(leonardoConfig);
 
-document.querySelector(".themes-alpha").innerHTML =
-  Object.values(themesHtmlAlpha).join("");
+  document.querySelector(".key-colors").innerHTML = keyColorsHtml;
 
-document.querySelector(".tokens-studio").innerHTML = tokensStudioHtml;
+  document.querySelector(".themes").innerHTML =
+    Object.values(themesHtml).join("");
 
-addStylesToHead(cssVars);
-document.querySelector(".css-vars__output").innerHTML = cssVars;
+  document.querySelector(".themes-alpha").innerHTML =
+    Object.values(themesHtmlAlpha).join("");
+
+  document.querySelector(".tokens-studio").innerHTML = tokensStudioHtml;
+
+  addStylesToHead(cssVars);
+  document.querySelector(".css-vars__output").innerHTML = cssVars;
+
+  document.querySelector(".leonardo-editor__links").innerHTML = leonardoHtml;
+
+  document
+    .querySelector(".key-colors__picker, #color-picker")
```

Comment:
```suggestion
    .querySelector("#color-picker")
```

Since that should be the only element firing the `input` event

Reply:
Ok, makes sense :)